src/sentry/apidocs/decorators.py

```diff
@@ -8,7 +8,7 @@
 def public(methods: HTTP_METHODS_SET) -> Callable[[Any], Any]:
     def decorate(view_cls: Endpoint) -> Endpoint:
 
-        PUBLIC_ENDPOINTS[view_cls.__name__] = {
+        PUBLIC_ENDPOINTS[f"{view_cls.__module__}.{view_cls.__name__}"] = {
```

Comment:
Is there any advantage to having the `PUBLIC_ENDPOINTS` keys be strings? If we're applying this same kind of formatting string in every case, could we just use the method object as a key instead?

Or is it not possible to recover the equivalent method object from the "callback" value passed to `custom_preprocessing_hook`? I admit those callback objects mystified me a bit when I attempted to get creative with metaprogramming.

Reply:
Yeah, I think it would be possible to grab the view class on the callback. Given that we're no longer using a decorator and the exclusion list registry will eventually be deprecated, going to leave off seeing if this approach works.